code/_helpers/global_lists.dm

```diff
@@ -58,7 +58,7 @@ var/datum/category_collection/underwear/global_underwear = new()
 
 	//Backpacks
 var/global/list/backbaglist = list("Nothing", "Backpack", "Satchel", "Satchel Alt", "Messenger Bag")
-var/global/list/pdachoicelist = list("Default", "Slim", "Old", "Rugged")
+var/global/list/pdachoicelist = list("ARFS", "Default", "Slim", "Old", "Rugged")//ARFS edit - For ceciPDAs

```

Comment:
This is a global variable and can just be overridden by writing the following into a new file.
pdachoicelist = list("ARFS", "Default", "Slim", "Old", "Rugged")